v3/src/components/codap-component.tsx

```diff
@@ -14,17 +14,30 @@ export interface IProps extends ITileBaseProps {
   Component: React.ComponentType<ITileBaseProps>;
   tileEltClass: string;
   onCloseTile: (tileId: string) => void
+  onBottomRightPointerDown: (e: React.PointerEvent) => void
+  onBottomLeftPointerDown: (e: React.PointerEvent) => void
+  onRightPointerDown: (e: React.PointerEvent) => void
+  onBottomPointerDown: (e: React.PointerEvent) => void
+  onLeftPointerDown: (e: React.PointerEvent) => void
```

Comment:
Make these optional since they won't all be appropriate for other layouts, e.g. mosaic
```suggestion
  onBottomRightPointerDown?: (e: React.PointerEvent) => void
  onBottomLeftPointerDown?: (e: React.PointerEvent) => void
  onRightPointerDown?: (e: React.PointerEvent) => void
  onBottomPointerDown?: (e: React.PointerEvent) => void
  onLeftPointerDown?: (e: React.PointerEvent) => void
```